airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/pipeline.py

```diff
@@ -243,6 +244,7 @@ async def update_package_version(self) -> StepResult:
         self.repo_dir = await dagger_write_file(repo_dir, file_path, new_content)
 
         if self.export:
+            assert self.repo_dir is not None
```

Comment:
this is so annoying because I make sure it's not none above. and my test pass locally? (`poetry run mypy pipelines --disallow-untyped-defs`)